CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.2]
+
+### Changed
+
+- Bump `@metamask/snaps-sdk` version to ^3.1.1 ([#271](https://github.com/MetaMask/keyring-api/pull/271))
```

Comment:
Maybe remove the backticks on this one, since it seems we're not doing this for the others and not for the prior versions too?